packages/vite/src/node/cli.ts

```diff
@@ -1,5 +1,5 @@
 import { cac } from 'cac'
-import colors from 'picocolors'
+import c from 'picocolors'
```

Comment:
Not sure if this refactoring is needed in this PR
I prefer **not** using abbreviations :thinking:

Reply:
In general I would agree but in this case the concatenation of `colors` call took a lot of place (since `picocolors` doesn't support chaining like `chalk`). 

I reverted the change to use `colors` again though 👍